components/dashboard/src/admin/TeamDetail.tsx

```diff
@@ -0,0 +1,105 @@
+/**
+ * Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import moment from "moment";
+import { useEffect, useState } from "react";
+import { Team, TeamMemberInfo, TeamMemberRole } from "@gitpod/gitpod-protocol";
+import { getGitpodService } from "../service/service";
+import { Item, ItemField, ItemsList } from "../components/ItemsList";
+import PillLabel from "../components/PillLabel";
+import DropDown from "../components/DropDown";
+import { Link } from "react-router-dom";
+
+export default function TeamDetail(props: { team: Team }) {
+    const { team } = props;
+    const [teamMembers, setTeamMembers] = useState<TeamMemberInfo[] | undefined>(undefined);
+    const [searchText, setSearchText] = useState<string>('');
+
+    useEffect(() => {
+        (async () => {
+            const members = await getGitpodService().server.adminGetTeamMembers(team.id);
+            if (members.length > 0) {
+                setTeamMembers(members)
+            }
+        })();
+    }, [team]);
+
+    const filteredMembers = teamMembers?.filter(m => {
+        const memberSearchText = `${m.fullName || ''}${m.primaryEmail || ''}`.toLocaleLowerCase();
+        if (!memberSearchText.includes(searchText.toLocaleLowerCase())) {
+            return false;
+        }
+        return true;
+    });
+
+    const setTeamMemberRole = async (userId: string, role: TeamMemberRole) => {
+        await getGitpodService().server.adminSetTeamMemberRole(team!.id, userId, role);
+        setTeamMembers(await getGitpodService().server.adminGetTeamMembers(team!.id));
+    }
+    return <>
+        <div className="flex">
+            <div className="flex-1">
+                <div className="flex"><h3>{team.name}</h3>
+                    {team.markedDeleted && <span className="mt-2"><PillLabel type="warn" className="font-semibold mt-2 py-0.5 px-2 self-center">Deleted</PillLabel></span>}
+                </div>
+                <p className="mb-6">/t/{team.slug}</p>
+                {!team.markedDeleted && <>
+                    <span className="text-gray-400">{teamMembers && teamMembers.length} member(s)</span>
+                    <span className="text-gray-400"> · </span>
+                </>}
+                <span className="text-gray-400">Created on {moment(team.creationTime).format('MMM D, YYYY')}</span>
```

Comment:
**suggestion**: What do you think of moving the create date metadata near the URL slug line? We could also skip members count in the detail page for now. Long-term I'd love to have one pattern that repeats consisntenly whteher this includes the URL slug or not.

<img width="340" alt="Screenshot 2022-02-24 at 12 27 02 PM" src="https://user-images.githubusercontent.com/120486/155507657-b4f82007-ede5-47af-9f52-ad43b9e2d45c.png">